README.md

```diff
@@ -13,6 +13,9 @@ Note that [auto-multiple-choice](https://www.auto-multiple-choice.net) (amc) LaT
 
 ## Installation
 
+A Docker image with `amc2moodle` and its dependencies is available at [ghcr.io/XXX/amc2moodle](ghcr.io/XXX/amc2moodle).
```

Comment:
Perhaps we should convert all  `ghcr.io/XXX/amc2moodle` to `ghcr.io/nennigb/amc2moodle`

Reply:
if everything is working within your github account, we can do it.

Reply:
Normally this is the case